sdk/storage/azure-storage-blob/azure/storage/blob/_shared/base_client.py

```diff
@@ -62,6 +62,7 @@
     "blob": {"primary": "BlobEndpoint", "secondary": "BlobSecondaryEndpoint"},
     "queue": {"primary": "QueueEndpoint", "secondary": "QueueSecondaryEndpoint"},
     "file": {"primary": "FileEndpoint", "secondary": "FileSecondaryEndpoint"},
+    "dfs": {"primary": "BlobEndpoint", "secondary": "BlobSecondaryEndpoint"},
 }
```

Comment:
dfs should not be needed here, right? No changes should happen to the blob package

Reply:
I think  this will be used by from_connection_string, however probably secondary is not necessary?